var/spack/repos/builtin/packages/octave/package.py

```diff
@@ -293,6 +305,9 @@ def configure_args(self):
         if spec.satisfies('%fj'):
             config_args.append('--enable-fortran-calling-convention=gfortran')
 
+        # Make sure we do not use qtchooser
+        config_args.append('ac_cv_prog_ac_ct_QTCHOOSER=')
```

Comment:
```py
        if not is_system_path(self.spec['qt'].prefix):
            config_args.append('ac_cv_prog_ac_ct_QTCHOOSER=')
```
@cessenat: @mtmiller didn't respond for 14 days after alalazo requesting his review.

Pending an eventual comment from him: Yes it looks sensible to only set `ac_cv_prog_ac_ct_QTCHOOSER=` when a `qt` built in spack is used and not externally.

Since `man qtchooser` says:
> The Qt Chooser provides a wrapper to switch between versions of Qt development binaries when multiple versions like 4 and 5 are installed or local Qt builds are to be used. It is commonly used via a symlink from an executable_name like qmake.

From reading this PR and the man `qtchooser` manual, it looks like qtchooser is a tool to select or run a specific Qt version or tool. As it looks like it is optional and not needed, I'd therefore think that it should be ok to always set `ac_cv_prog_ac_ct_QTCHOOSER` empty, but I didn't check in detail what the `configure` script does based on it available or not.

Unless @mtmiller answers soon, would you maybe check what the `configure` script does and unless you reach a clear conclusion, set it empty always? When you are ready to merge this PR please notify me if you the button is green for you now, you can merge directly.

Reply:
Yes @bernhardkaindl I think you are right, we can always set it empty since when building with spack we want some kind of reproducibility so when building octave using an external qt, one will be specific about the qt version, i.e. there should be no need of a qtchooser. Let us wait up to nov, 8th for @mtmiller advice.